q2lint.py

```diff
@@ -27,6 +28,19 @@ def main():
             if header != HEADER:
                 errors.append('Invalid header: %s' % filepath)
 
+    npm_packages = filter(lambda x: 'node_modules' not in x,
+                          glob.glob('**/*/package.json', recursive=True))
+    if npm_packages:
+        import subprocess
+        import os
+
+        for package in npm_packages:
+            pkg_path, _ = os.path.split(package)
+            cmd = 'cd %s; npm i; npm run build; git diff --quiet */bundle.js' \
```

Comment:
Should these commands be joined with `&&` instead of `;`? Using `&&` will short-circuit on the first failing command and propagate that command's error code.